server/app/views/components/AlertFragment.html

```diff
@@ -0,0 +1,36 @@
+<div
+  th:fragment="alert(alertSettings, headingLevel)"
```

Comment:
Note: I just could not get this to let me pass in an enum. Not adding the headingLevel to the AlertSettings class since setting the heading is definitely a UI thing and doesn't need to be thought about when making the AlertSettings in a service or controller.

Reply:
greg ran into issues passing enums too

Reply:
If it's part of an object it works